### ISSUE TITLE
WearOS app freezing on start

### ISSUE THREAD

**Issue by livagit** [opengapps/opengapps#910]
I think there is something with opengapps. I have Xiaomi Redmi Note 5 Pro (whyred), last build 17.1 lineage-17.1-20210222-nightly-whyred, open_gapps-arm64-10.0-nano-20210220(20210213) and the same errors:
https://forum.xda-developers.com/t/rom-official-lineageos-17-1-weeklies-for-xiaomi-mi-mix-2s.3974661/post-84473935
I installed opengapps nano 20210206, updated it (Google, Google Play services, WearOS) from the market and the WearOS application starts. And right now  I connected a smartwatch. )

**Comment by rk-zero** [comment]
Pair the watch from bluetooth settings, its broken inside WearOS App. Continue setup from the notification prompt that follows after pairing / starting a fresh wiped watch.

**Comment by livagit** [comment]
OMG. It all started with the fact that I could not open the application on the already paired watches after the next upgrade of LOS and ogapps (may be Wear OS). Well, now I have all the updates installed from the market on what I installed from the recovery (see above) and so far Wear OS is opening. But with the next upgrade LOS I will  try to update and ogapps too.

**Comment by rk-zero** [comment]
Yeah its a huge clusterf.... if you have a paired watch and backup of wearos it should work around that. But the issue seems broken bluetooth managing in wearos/playservices.

**Comment by tsjk** [comment]
Is there a way to start Wear OS in that mode without the notification? For some reason I don't receive it. I have reinstalled my phone and reset my watch. While I do have a backup of the stuff on my phone from before the re-installation, the watch reset is pretty much a no-way-back thing.

**Comment by copperly123** [comment]
I'm having this issue too, I cannot pair my watch on my Moto X4 running the latest versions of Lineage OS 17.1 and Open Gapps nano. The wear OS app gets stuck on the splash screen, I can pair my watch via the bluetooth settings and the notification to set up my watch does pop up but it only brought up the wear os app with half of the options missing and only allowed me to give it some permissions.

Wear OS was installed through a backup using Swift Backup, but I have since cleared storage on google play services, Wear OS, and the Google app, and I reinstalled the Wear OS app. I factory reset my watch, which is a Ticwatch E
